server/app/dal/CassandraCluster.scala

```diff
@@ -43,19 +43,9 @@ class CassandraConfig @Inject()(configuration: Configuration, environment: Envir
       config.getStringList(hostsConf)
     }
 
-    def docker: Option[Boolean] = configuration.getBoolean(dockerConf)
-
-    def dockerIp() = docker match {
-      case Some(true) =>
-        if (environment.mode == Mode.Prod)
-          Logger.error(s"$dockerConf = true, and we are in prod mode")
-        Try(Seq("docker-machine", "ip", "default").!!.trim).toOption
-      case _ => None
-    }
-
-    dockerIp().fold(hosts) { ip =>
-      Logger.warn(s"$hostsConf overridden with docker $ip")
-      Seq(ip)
+    val r = new Regex( """(\$\()([^)]*)(\))""", "$(", "command", ")")
```

Comment:
Why do you need regexp here at all? Why don't execute what is inside "hosts" variable ?